### PR TITLE
remove WNPs 119-124

### PR DESCRIPTION
**Pull request by wen-2018**
## One-line summary

This PR removes WNPs 119-124.

- [ ] ~I used an AI to write some of this code.~

## Issue / Bugzilla link

Fixes https://github.com/mozilla/bedrock/issues/14661

### CODE REVIEW

File: bedrock/firefox/tests/test_base.py
Comment (janbrasna) [inline bedrock/firefox/tests/test_base.py:502]: I think this is what `ruff` [dislikes](https://github.com/mozilla/bedrock/actions/runs/9472271392/job/26097370005?pr=14670#step:4:6050) — there will be two consecutive empty lines. I guess if you just remove the additional new line, it would pass.

**Comment by janbrasna** [comment]
The conflict with merged [#14644](https://github.com/mozilla/bedrock/pull/14644/files#diff-aa470d557b9b54f147f88cf05dd8b3a95f7e2d24f6f28af69646469e95d88c53) should be an easy fix to resolve.

**Review by reemhamz — APPROVED** [review]
Seamlessly removed everything! 🏅 
I do however notice that there are mentions of whatsnew120-122 in [this redirects file](https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/redirects.py#L29-L34), I think they're safe to remove but I'll defer to @alexgibson or @craigcook here for confirmation before we merge

**Comment by janbrasna** [comment]
Even old links/campaigns keep working with these gone — e.g. with "firefox-whatsnew-121-us" removed, there's a greedy "firefox-whatsnew" catchall matching the beginning of the passed query string that would pick at least some attribution as a fallback. (You can try with bogus value here https://www-dev.allizom.org/firefox/browsers/mobile/app/?product=firefox&campaign=firefox-whatsnew-129-dummy redirecting to app store still with at least some attribution.)

**Comment by alexgibson** [comment]
@reemhamz if the pages that feature these QR codes are gone I'd say it's safe to remove those campaign entries from the redirects file. As @janbrasna says, they will fall back to `firefox-whatsnew` 👍

**Comment by reemhamz** [comment]
I noticed one more image that needs removing: https://github.com/mozilla/bedrock/blob/main/media/img/l10n/en-US/firefox/wnp-121/qrcode-firefox-whatsnew-121.svg

**Comment by wen-2018** [comment]
Good catch! It's been removed now

**Pull request merged**
